Gemfile.lock

```diff
@@ -1,5 +1,7 @@
-PATH
-  remote: ../maze-runner
+GIT
+  remote: https://github.com/bugsnag/maze-runner
+  revision: b12f25a65267a99162a40abf28fe02c3048a097f
+  tag: v4.10.1
```

Comment:
Is this supposed to change?

Reply:
I believe so - the `Gemfile` was bumped to `v4.10.1` but the lockfile wasn't updated. This is generated as a result of running `bundle install` which is a prerequisite of running the local E2E tests.